jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -470,8 +481,7 @@ else if (args.isCreateFiles() || !args.getStartModules().isEmpty())
             CommandLineBuilder cmd = args.getMainArgs(StartArgs.ALL_PARTS);
             cmd.debug();
 
-            List<String> execModules = args.getEnabledModules().stream()
-                .map(name -> args.getAllModules().get(name))
+            List<String> execModules = args.getValidEnabledModulesStream()
```

Comment:
Direct prevention of the NPE.
This is preventative, ideally we shouldn't be able to reach here with bad module names, as it should error out with a UsageException for modules that don't exist earlier.

Reply:
This feels a bit artificial.  I'd prefer that we just modified `getEnabledModules` to never return null.

Reply:
This should have been called something else, "getSelectedModuleNames" or "getRequestedModuleNames".
It's not the list of enabled modules, never has been.
We don't even update this list for full list of modules that includes the transitive modules.
That's what the `Modules` object is for.

Reply:
The new method `StartArgs.getValidEnabledModulesStream() `is no longer present.